pkgs/data/fonts/nerdfonts/create-shas.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# NOTE: Before running this script, please make sure this list is up-to-date -
+# meaning there are no new fonts they provide
+
+nerdfonts_version="2.1.0"
```

Comment:
```suggestion
nerdfonts_version=$(nix eval --raw nixpkgs.nerdfonts.version}
```

Reply:
This one is a bit more challenging - wouldn't `nix eval` print the version currently defined?

Reply:
@doronbehar Couldn't you just update the expression and then run the script?
That sounds like a sensible flow to me.

Though you'll have to pass the current nixpkgs checkout with `-f`, since this will use the nixpkgs of the system.

Reply:
I improved the script even further - it updates the version in a `version.nix` after it retrieves the latest version from their releases page.